identity_iota_core/src/rebased/iota/package.rs

```diff
@@ -0,0 +1,226 @@
+// Copyright 2020-2025 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+#![allow(dead_code)]
+
+use std::collections::HashMap;
+use std::sync::LazyLock;
+
+use anyhow::Context;
+use identity_iota_interaction::types::base_types::ObjectID;
+use serde::Deserialize;
+use serde::Deserializer;
+use serde::Serialize;
+use tokio::sync::RwLock;
+use tokio::sync::RwLockReadGuard;
+use tokio::sync::RwLockWriteGuard;
+use tokio::sync::TryLockError;
+
+use crate::rebased::client::CoreClientReadOnly;
+use crate::rebased::Error;
+
+pub(crate) const MAINNET_CHAIN_ID: &str = "6364aad5";
+
+type PackageRegistryLock = RwLockReadGuard<'static, PackageRegistry>;
+type PackageRegistryLockMut = RwLockWriteGuard<'static, PackageRegistry>;
+
+static IDENTITY_PACKAGE_REGISTRY: LazyLock<RwLock<PackageRegistry>> = LazyLock::new(|| {
+  let move_lock_content = include_str!(concat!(env!("CARGO_MANIFEST_DIR"), "/packages/iota_identity/Move.lock"));
```

Comment:
As we're compiling for windows as well in the CI, should we consider Windows paths here as well?

> The provided path is interpreted in a platform-specific way at compile time. So, for instance, an invocation with a Windows path containing backslashes \ would not compile correctly on Unix.

[`include_str!`](https://doc.rust-lang.org/std/macro.include_str.html)